src/Core/View.js

```diff
@@ -509,4 +510,82 @@ View.prototype.normalizedToViewCoords = function normalizedToViewCoords(ndcCoord
     return _eventCoords;
 };
 
+/**
+ * Return objects from some layers/objects3d under the mouse in this view.
+ *
+ * @param {Object} mouse - mouse position in window coordinates (0, 0 = top-left)
+ * @param {...*} where - where to look for objects. Can be either: empty (= look
+ * in all layers with type == 'geometry'), layer ids or layers or a mix of all
+ * the above.
+ * @return {Array} - an array of objects. Each element contains at least an object
+ * property which is the Object3D under the cursor. Then depending on the queried
+ * layer/source, there may be additionnal properties (coming from THREE.Raycaster
+ * for instance).
+ *
+ * @example
+ * view.pickObjectsAt({ x, y })
+ * view.pickObjectsAt({ x, y }, 'wfsBuilding')
+ * view.pickObjectsAt({ x, y }, 'wfsBuilding', myLayer)
+ */
+View.prototype.pickObjectsAt = function pickObjectsAt(mouse, ...where) {
+    const results = [];
+    const source = where.length == 0 ?
+        this.getLayers(l => l.type == 'geometry') :
+        [...where];
+
+    for (let i = 0; i < source.length; i++) {
+        if (typeof (source[i]) === 'string') {
+            const lookup = this.getLayers(l => l.id == source[i]);
+            if (!lookup.length) {
+                throw new Error(`Invalid layer id used as where argument (value = ${where})`);
+            }
+            source[i] = lookup[0];
+        }
+    }
+
+    for (let i = 0; i < source.length; i++) {
+        if (source[i].type === 'geometry') {
+            // does this layer have a custom picking function?
+            if (source[i].pickObjectsAt) {
+                results.splice(
+                    results.length, 0,
+                    ...source[i].pickObjectsAt(this, mouse));
+            } else {
+                //   - it hasn't: this layer is attached to another one
+                let parentLayer;
+                this.getLayers((l, p) => {
+                    if (l.id == source[i].id) {
+                        parentLayer = p;
+                    }
+                });
+
+                const obj = [];
+                // raycast using parent layer object3d
+                Picking.pickObjectsAt(
```

Comment:
Sorry I don't understand your comment.

Reply:
forget the previous comment

Why geometry layer doesn't have default `pickObjectsAt` method?

Reply:
All GeometryLayer have a default picking method (see https://github.com/iTowns/itowns/pull/635/files#diff-61e012c3e00bdc510be33d9eac684b2aR80)

But the method in View support picking for all objects (for instance raw THREE.Object3D) so we need to use `Picking.pickObjectsAt` as a fallback.

Reply:
> for instance raw THREE.Object3D 

Should be explain in the title commit because it doesn't unify the picking position/coordinates and features (in raster)